install

```diff
@@ -10,6 +10,8 @@ Icon=$HOME/pi-ware/icon.png
 Categories=Utility;
 Type=Application
 Terminal=false" > $HOME/.local/share/applications/pi-ware.desktop
+echo "Installing LXTerminal"
+sudo apt install lxterminal 
```

Comment:
Add `-y` to the line that installs lxterminal so apt doesn't prompt for confirmation.

Reply:
Thanks! I’ll do that now. Feel free to request an app in an issue.

Reply:
So sudo apt install lxterminal -y?

Reply:
yes